muxfs/mux.go

```diff
@@ -50,6 +57,7 @@ var constructors = map[string]qfs.FSConstructor{
 	"local": localfs.NewFilesystem,
 	"http":  httpfs.NewFilesystem,
 	"mem":   qfs.NewMemFilesystem,
+	"map":   cafs.NewMapFilesystem,
```

Comment:
Damn, didn't realize we had actual "map" and "mem" implementations.

Reply:
Yea, in Qri land we expect `mem` and `map`. `map` is used wherever we need a cafs filestore and `mem` is typically used for the repo. Also, the config validation only expects `map` as an option for `Store` and `mem` as an option for `Repo`.